coralproject/talk
# [CORL-371] Allow explicitly disabling client routes

## What does this PR do?
- The purpose of this is to avoid warnings like `WARN: client routes are disabled` during client development using Webpack Dev Server.

- Server now supports `DISABLE_CLIENT_ROUTES` env variable to explicitly disable client routes.
- Disable client routes when using `npm run watch` as we are using Webpack Dev Server for those routes instead.

## How do I test this PR?

- `npm run watch` and see much more subtle warning: "WARN: client routes are disabled"
